src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/AvslagBegrunnelser.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { useVedtakBegrunnelser } from '../../../../context/VedtakBegrunnelseContext';
+import { IBehandling } from '../../../../typer/behandling';
+import { IRestAvslagbegrunnelser } from '../../../../typer/vedtak';
+import { RessursStatus } from '@navikt/familie-typer';
+import { Vedtaksperiode, Vedtaksperiodetype } from '../../../../typer/vedtaksperiode';
+import familieDayjs, { familieDayjsDiff } from '../../../../utils/familieDayjs';
+import { datoformat } from '../../../../utils/formatter';
+import OverskriftMedHjelpetekst from './Felles/OverskriftMedHjelpetekst';
+import EkspanderbartBegrunnelsePanel from './Felles/EkspanderbartBegrunnelsePanel';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import styled from 'styled-components';
+
+interface IAvslagTabell {
+    åpenBehandling: IBehandling;
+}
+
+const PanelBody = styled.div`
+    ul {
+        margin: 0;
+        padding-left: 1.5rem;
+    }
+`;
+
+const AvslagBegrunnelser: React.FC<IAvslagTabell> = ({ åpenBehandling }) => {
+    const { avslagBegrunnelser } = useVedtakBegrunnelser();
+
+    return avslagBegrunnelser.status == RessursStatus.SUKSESS && avslagBegrunnelser.data.length ? (
+        <>
+            <OverskriftMedHjelpetekst
+                overskrift={'Begrunnelser for avslag i vedtaksbrev'}
+                hjelpetekst={
+                    'Her har vi hentet begrunnelsestekster for avslag som du har satt i vilkårsvurderingen.'
+                }
+            />
+            {åpenBehandling.vedtaksperioder
+                .filter(
+                    (periode: Vedtaksperiode) =>
+                        periode.vedtaksperiodetype === Vedtaksperiodetype.AVSLAG
+                )
+                .sort((a, b) =>
+                    !a.periodeFom && !a.periodeTom
```

Comment:
Er det riktig å sjekke for periodeTom her?

Reply:
Ja, det den gjør er å legge "tomme" perioder nederst, men jeg kan legge det i en egen funksjon for å tydeliggjøre!

Reply:
Ja, hvis ingen er satt så skal alle være sortert likt som fra APIet antar jeg?